.github/workflows/test-with-edm.yml

```diff
@@ -9,9 +11,47 @@ env:
 
 jobs:
 
-  # Test against EDM packages
-  # Note that some packages may not actually be installed from EDM but from
-  # PyPI, see ci/edmtool.py implementations.
+  # Test against EDM packages on Linux
+  test-edm-linux:
+    strategy:
+      matrix:
+        toolkit: ['pyqt', 'pyqt5', 'pyside2', 'wx']
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Qt dependencies for Linux
+        run: |
+          sudo apt-get update
```

Comment:
Any specific reason why we are running `... update`? I'm asking because this adds a few extra seconds to the overall time the job takes to finish.

Reply:
It is included because I believe this is good practice to update the package repository before installing them. If it becomes a bottleneck we may be able to omit it.